shuup_tests/browser/notify/test_script_template.py

```diff
@@ -53,33 +54,42 @@ def test_generic_script_template(browser, admin_user, live_server, settings, scr
 
     url = reverse("shuup_admin:notify.script.list")
     browser.visit("%s%s" % (live_server, url))
-    wait_until_condition(browser, lambda x: x.is_element_present_by_css("div.btn-toolbar a.btn.btn-info"))
+    wait_until_appeared(browser, "div.btn-toolbar a.btn.btn-info")
+    # wait_until_condition(browser, lambda x: x.is_element_present_by_css("div.btn-toolbar a.btn.btn-info"))
 
     # find the button to load from template
     browser.find_by_css("div.btn-toolbar a.btn.btn-info").first.click()
 
     identifier = script_template_cls.identifier
     form_id = "form-" + identifier
     button_id = "#{} button.btn.btn-success".format(form_id)
-    wait_until_condition(browser, lambda x: x.is_element_present_by_css(button_id))
+    wait_until_appeared(browser, button_id, timeout=20)
     browser.find_by_css(button_id).first.click()
 
     config_url = reverse("shuup_admin:notify.script-template-config", kwargs={"id": identifier})
     wait_until_condition(browser, lambda b: b.url.endswith(config_url), timeout=15)
     wait_until_condition(browser, lambda b: b.is_text_present("Configure the Script Template"))
 
     # click to create the script
+    time.sleep(5)
```

Comment:
Instead of randomly sleeping here maybe we should have some util for executing scripts. That method could also do the same thing for all the scripts.